README.md

```diff
@@ -140,23 +140,86 @@ sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/ssl/privat
+
+     ```bash
+     git clone https://github.com/habitat-sh/on-prem-builder.git
+     export DOWNLOAD_DIR=/some/base/download/directory
+     cd on-prem-builder
+     hab pkg download --target x86_64-linux --channel stable --file package_seed_lists/builder_x86_64-linux_stable --download-directory ${DOWNLOAD_DIR}/builder_packages
+     ```
+
+1. Create any additional package bundles to upload to Builder from package seed lists as documented in the [Bootstrap Builder](https://github.com/habitat-sh/on-prem-builder/blob/master/README.md#bootstrap-builder-with-habitat-packages-new) section of this README. You can specify `--download-directory ${DOWNLOAD_DIR}/builder_bootstrap` argument to the download command in order to consolidate all bootstrap packages in a single directory
+1. Zip up all the above content, transfer and unzip on the Linux system where Builder will be deployed in the Airgapped environment
+
+> Note: The following tasks are intended to be completed on the Airgapped system where Builder will be deployed, in advance of the installation
+
+1. From the zip archive, install the `hab` binary somewhere in $PATH and ensure it has execute permissions:
+
+     ```bash
+     sudo chmod 755 /usr/bin/hab
+     sudo hab # read the license and accept if in agreement, as the root user
+     ```
+
+1. Import the public package signing keys from the downloaded Builder package bundle:
+
+     ```bash
+     export UNZIP_DIR=/some/base/unzip/directory
+     for file in $(ls ${UNZIP_DIR}/builder_packages/keys/*pub); do cat $file | sudo hab origin key import; done
+     ```
+
+1. Create a Habitat artifact cache directory, place the Builder `*.hart` packages into that directory and then pre-install the Builder Services:
+
+     ```bash
+     sudo mkdir -p /hab/cache/artifacts
+     sudo mv ${UNZIP_DIR}/builder_packages/artifacts/*hart /hab/cache/artifacts
+     sudo hab pkg install /hab/cache/artifacts/habitat-builder*hart
+     ```
+
+1. Pre-install the Habitat Supervisor and its dependencies:
+
+     ```bash
+     sudo hab pkg install --binlink --force /hab/cache/artifacts/core-hab-*hart
+     ```
+
 ## Setup
 
-1. Clone this repo (or unzip the archive you have downloaded from the Github release page) at the desired machine where you will stand up the Chef Habitat Builder on-prem
+1. Clone this repo (or unzip the zip archive you previously downloaded from the Github release page) at the desired machine where you will stand up the Chef Habitat Builder on-prem
 1. `cd ${SRC_ROOT}`
 1. `cp bldr.env.sample bldr.env`
 1. Edit `bldr.env` with a text editor and replace the values appropriately. Consider helping us to improve Chef Habitat as well by changing the `ANALYTICS_ENABLED` setting to `true` and providing an optional company name.
+
+## Installation
```

Comment:
Does this section apply for both airgap as well as non-airgap?

Reply:
Yes, it is the same for airgap and non-airgap.